R/data.R

```diff
@@ -33,6 +33,16 @@
 #'
 "admiral_adsl"
 
+#' Lab Analysis Dataset
+#'
+#' An example of lab analysis dataset
+#' @keywords datasets
+#' @family datasets
+#' @source
+#' Derived from the `adlb` template.
+#'
+"admiral_adlb"
+
```

Comment:
Is there code for this?

Reply:
It was originally, ***Run the script in use_ad_template("adlb") then add:
```
admiral_adlb <- adlb
usethis::use_data(admiral_adlb)
```

but then I added the `filter(USUBJID %in% c(...)) ` for the 11 patients to reduce the size of the dataset then overwrote the original object I had initially uploaded

Reply:
Can that possibly be inserted into the @source that you filtered down to those patients.